artifactory.py

```diff
@@ -64,6 +64,10 @@
 default_config_path = "~/.artifactory_python.cfg"
 global_config = None
 
+# set logger to be configurable from external
```

Comment:
The best practice is to define a library's logger and then use it everywhere in the code.

It does use default handlers, so there's no needs to set it to NullHandler

```python
logger = logging.getLogger(__name__)

# use it later
logger.info("INFO")
logger.debug("DEBUG")
```

Reply:
there is a need, see comment in requests

https://github.com/psf/requests/blob/1e5fad7433772b648fcbc921e2a79de5c4c6be8b/requests/__init__.py#L145

if logger was not configured, we do not want to dump any message to the user. We have to use NullHandler

Reply:
Make sense. Let's leave the comment as in the link and merge the PR
 `# Set default logging handler to avoid "No handler found" warnings.`

Reply:
done